errai-data-binding/src/test/java/org/jboss/errai/databinding/client/TestModelWithIgnoreAnnotation.java

```diff
@@ -0,0 +1,44 @@
+package org.jboss.errai.databinding.client;

```

Comment:
Please add a license header to this file. (You can copy it from any of the other Java source files in this project, but remember to change the year.)